census/census.go

```diff
@@ -65,15 +76,30 @@ type Manager struct {
 	failedQueueLock sync.RWMutex
 	failedQueue     map[string]string
 	compressor
+	cancel context.CancelFunc
+	wg     sync.WaitGroup
```

Comment:
I'd give this a better name, like `wgQueueDaemons`, to clarify what it's used for